Makefile

```diff
@@ -3,6 +3,7 @@ SRCS_JS_FROM_TS =
 
 SRCS_TS += index.ts
 SRCS_TS += $(shell find lib -type f -name '*.ts')
+SRCS_TS += $(shell find test -type f -name '*.ts')
```

Comment:
I tried to achieve what you described over the weekend but no luck, will appreciate your help! @ericvw